src/components/entity/EntityResult.vue

```diff
@@ -53,7 +53,42 @@
           <div class="mb-6 text-body-2">
             <span>{{ entityName }}</span>
             <span>&nbsp;is a&nbsp;</span>
-            <span class="fake-link" v-t="entityJurisdiction"></span>
+            <Dialog>
```

Comment:
Im not opposed to moving these into their own component files so the parent component remains slim. Up to you though

Reply:
eventually I think we'll be moving the text into the database if we're going to do multilingual support. I think it's okay here for now until we're ready to make the change